content/pandas/concepts/built-in-functions/terms/read-excel/read-excel.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.read_excel()'
+Description: 'Reads an Excel file into a Pandas DataFrame object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Data Structures'
+  - 'Functions'
+  - 'Pandas'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The `.read_excel()` function takes a path to a Excel file and reads the data into a Pandas [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) object.
+
+## Syntax
+
+```pseudo
+pandas.read_excel(io, sheet_name)
+```
+
+The `io` parameter is the path to the Excel file. It can be a path on the local machine, or a valid URL. It is the first parameter of the function, and can be used by itself.
```

Comment:
```suggestion
The `io` parameter is the path to the Excel file. It can be a path on the local machine or a valid URL. It is the first parameter of the function and can be used by itself.
```